### PR TITLE
iconnection

### LINKED ISSUE #1327: IConnection Link not working
When clicking on this link: https://docs.dataminer.services/develop/api/types/Skyline.DataMiner.Net.IConnection.html
(e.g. available in Method [GetDms](https://docs.dataminer.services/develop/api/types/Skyline.DataMiner.Core.DataMinerSystem.Common.ConnectionInterfaceExtensions.GetDms.html)), the page is not found (404)

### PR DESCRIPTION
**Pull request by PedroDebevere**
Fixes #1327

### CODE REVIEW

File: src/DataMiner/SLNetTypes/SubscriptionFilters/ITrackedSubscriptionUpdate.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+		ITrackedSubscriptionUpdate OnFinished(Action action);
+
+		/// <summary>
+		/// Executes this instance.
+		/// </summary>
+		/// <returns>DMS messages.</returns>
+		DMSMessage[] Execute();
+
+		/// <summary>
+		/// Executes the and wait.
```
Comment (MariekeGO) [inline src/DataMiner/SLNetTypes/SubscriptionFilters/ITrackedSubscriptionUpdate.cs:51]: Something seems to be missing here

File: src/DataMiner/SLNetTypes/SubscriptionFilters/SubscriptionFilter.cs [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+	///			SubscriptionFilterCluster: limits subscription to one particular cluster 
+	///		</description></item>
+	///		<item><description>
+	///			SubscriptionFilterDMA: limits messages to one particular DataMiner Agent
+	///		</description></item>
+	///		<item><description>
+	///			SubscriptionFilterElement: limits messages to one particular element
+	///		</description></item>
+	///		<item><description>
+	///			SubscriptionFilterParameter: limits messages to one particular parameter of a given parameter
```
Comment (MariekeGO) [inline src/DataMiner/SLNetTypes/SubscriptionFilters/SubscriptionFilter.cs]: Is this correct? "a parameter of a parameter" seems a bit odd.

File: src/DataMiner/SLNetTypes/SubscriptionFilters/SubscriptionFilter.cs [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+		{
+		}
+
+		protected SubscriptionFilter(SubscriptionFilter subscriptionFilter)
+			: this(subscriptionFilter.MessageType, subscriptionFilter.Filters)
+		{
+		}
+
+		/// <summary>
+		/// The message type about which the <see cref="SubscriptionFilter"/> handles.
```
Comment (MariekeGO) [inline src/DataMiner/SLNetTypes/SubscriptionFilters/SubscriptionFilter.cs:90]: This isn't quite clear. Could you rephrase it?

File: src/DataMiner/SLNetTypes/SubscriptionFilters/SubscriptionFilter.cs [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+		/// </summary>
+		/// <value>The extra options.</value>
+		public SubscriptionFilterOptions Options { get; set; }
+
+		/// <summary>
+		/// Determines whether the specified option is present.
+		/// </summary>
+		/// <param name="option">The option.</param>
+		/// <returns>
+		///   <c>true</c> if the specified option has option; otherwise, <c>false</c>.
```
Comment (MariekeGO) [inline src/DataMiner/SLNetTypes/SubscriptionFilters/SubscriptionFilter.cs:183]: Is this correct?

File: src/DataMiner/SLNetTypes/SubscriptionFilters/SubscriptionFilter.cs [hunk 5]
```diff
@@ -0,0 +1,268 @@
...
+
+		/// <summary>
+		/// Base namespace for messages. If message types are within this namespace,
+		/// they can be qualified with their type name only, without the need
+		/// to specify the full type namespace.
+		/// </summary>
+		public const string BaseNameSpace = "Skyline.DataMiner.Net.Messages.";
+
+		/// <summary>
+		/// Message type of which all messages must derive.
```
Comment (MariekeGO) [inline src/DataMiner/SLNetTypes/SubscriptionFilters/SubscriptionFilter.cs:250]: This seems a bit odd. Did you mean this?

```suggestion
		/// Message type from which all messages must derive.
```

File: src/DataMiner/SLNetTypes/SubscriptionFilters/SubscriptionFilter.cs (hunk 3 above)
Comment (PedroDebevere) [reply src/DataMiner/SLNetTypes/SubscriptionFilters/SubscriptionFilter.cs]: updated

File: src/DataMiner/SLNetTypes/SubscriptionFilters/SubscriptionFilter.cs (hunk 2 above)
Comment (PedroDebevere) [reply src/DataMiner/SLNetTypes/SubscriptionFilters/SubscriptionFilter.cs]: updated

File: src/DataMiner/SLNetTypes/SubscriptionFilters/ITrackedSubscriptionUpdate.cs (hunk 1 above)
Comment (PedroDebevere) [reply src/DataMiner/SLNetTypes/SubscriptionFilters/ITrackedSubscriptionUpdate.cs]: updated

**Review by MariekeGO — APPROVED** [review]
